### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by zhly0** [zisianw/FaceBoxes.PyTorch#33]
what is the training loss look like？how many epochs have you trained and what is the loss value？I trained it for about one day in a 1080ti machine，x 0.1  learning rate in the middle of the day，but the loss for L is 2.5，the loss for C is 1.5，is  the loss too big？

**Comment by zisianw** [comment]
Hi @zhly0, I remember I had similar number as yours too during training, I think it's fine.

**Comment by zhly0** [comment]
@zisianw，Thanks。
